File: IT/Input-Output/C++/readFile.cpp
/* file readFile.cpp */

#include <iostream>
#include <fstream>

using std::ifstream;
using std::string;
using std::cout;
using std::endl;

int main()
{
	/* apre il file */

	ifstream file1("helloworld.txt");

	string s;

	/* stampa il contenuto del file */

	while (file1.good())
	{
		getline(file1, s);
		cout << s;
	}

	cout << endl;

	/* chiude il file */

	file1.close();
}

File: IT/Input-Output/Java/ReadFile.java
/* file ReadFile.java */

import java.io.File;
import java.io.FileReader;
import java.io.BufferedReader;
import java.io.IOException;

public class ReadFile {
	public static void main(String [] args) {
		try {
			/* apre il file */

			File file1 = new File("helloworld.txt");
			FileReader reader = new FileReader(file1);
			BufferedReader input = new BufferedReader(reader);
		
			/* stampa il contenuto del file */

			while (input.ready()) {
				System.out.println(input.readLine());
			}

			/* chiude il file */

			reader.close();

		} catch (IOException e) {
			e.printStackTrace();
		}
	}
}

